tabulon_dxf/src/lib.rs

```diff
@@ -13,11 +13,13 @@ use tabulon::{
     render_layer::RenderLayer,
     shape::{AnyShape, FatPaint, FatShape},
     text::{AttachmentPoint, FatText},
-    DirectIsometry, GraphicsBag, GraphicsItem, ItemHandle,
+    DirectIsometry, GraphicsBag, GraphicsItem, ItemHandle, PaintHandle,
 };
 
 use parley::{Alignment, StyleSet};
 
+use joto_constants::u64::MICROMETER;
```

Comment:
Combine with parley imports in a single block, in alpha order. Might need doing in other places too.